### PR TITLE
ADDED:User_reg_form

### PR DESCRIPTION
**Pull request by PrakashVjr**
User  Registration Form Initial

### CODE REVIEW

File: Prakash/User_reg_form/style2.css [hunk 1]
```diff
@@ -0,0 +1,151 @@
+*{
+    padding: 0;
```
Comment (NavinAT) [inline Prakash/User_reg_form/style2.css:2]: Indent the code properly

**Review by NavinAT — CHANGES_REQUESTED** [review]
Reviewed your code please change and give the new pull request.

Please check the `<form>` tag path i couldn't run the project.
Please provide a proper run able solution then i can check the process

Thanks.

File: Prakash/User_reg_form/edit.html [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+            Edit Form
+        </h1>
+    </div>
+
+</head>
+<body>
+    <body>
+   
+        <div class="main">
+
```
[the remark this answers is not in the document]
Comment (PrakashVjr) [reply Prakash/User_reg_form/edit.html]: test

**Comment by PrakashVjr** [comment]
Resolved closed by naveen

**Pull request closed without merging**